packages/server/src/scim.ts

```diff
@@ -124,7 +124,10 @@ export class ScimServer implements DirectoryProvider {
     }
 
     private async _getScimOrg(orgId: OrgID) {
-        const prov = await this.storage.get(OrgProvisioning, orgId);
+        const prov = await this.storage.get(OrgProvisioning, orgId).catch(() => null);
```

Comment:
Why not wrap this in `try/catch` returning null instead? Like:

```suggestion
        try {
          const prov = await this.storage.get(OrgProvisioning, orgId);
        } catch (_error) {
          return null;
        }
```

Since that seems easier to read and maintain.

Reply:
Because it's 5 lines instead of 4 😄. JK, I have no preference either way. We can make that change, but then we should also remove the conditional return below.

Honestly I kind of regret having `Storage::get` throw and error instead of simply returning `null` if nothing is found, but it's a little late to change that now.

Reply:
Yes, I couldn't easily suggest removing the conditional below.